tests/test_roles.py

```diff
@@ -0,0 +1,45 @@
+import json
+import unittest
+import os, sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from fusillade.clouddirectory import Role, cleanup_directory, cleanup_schema
+from tests.common import new_test_directory
+directory = None
+schema_arn = None
+
+
+def setUpModule():
+    global directory, schema_arn
```

Comment:
Could these globals be set as an env var?

Reply:
fixed